From: Lee Duncan <lduncan@suse.com>
Date: Tue Jan 18 11:34:13 PST 2022
Subject: [PATCH] mhvtl kernel module: fix sysfs emit decl

Fix the sysfs_emit declaration so that kernels that don't
have this function will still compile. Tested on 5.3.18.

Will be submitted upstream.
---
diff --git a/kernel/backport.h b/kernel/backport.h
index 5c03f6e8150f..ef70aef20004 100644
--- a/kernel/backport.h
+++ b/kernel/backport.h
@@ -49,7 +49,7 @@ static inline struct inode *file_inode(struct file *f)
 #define HAVE_UNLOCKED_IOCTL 1
 #endif
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 10, 0)
 /* https://patches.linaro.org/project/stable/patch/20210305120853.392925382@linuxfoundation.org/ */
 /**
  *	sysfs_emit - scnprintf equivalent, aware of PAGE_SIZE buffer.
